### PR TITLE
Type changes

### LINKED ISSUE #64: [REFACTOR] Small changes needed to increase code readability and maintainability.
Small amounts of refactoring can be done to all files under `cmd` which have additional flags.

## Creating `const` types for default values of flags.
- Each flag in [cobra]() needs to have a default value set. As of now the default value in some cases is mentioned while creating the flag such as https://github.com/pesos/grofer/blob/03a98094cd2fd6b8822d0295f0e38e230e5ee730/cmd/proc.go#L106

- For increased readability and also maintainability, it would be better to have a `const` of these default values like [this](https://github.com/pesos/grofer/blob/03a98094cd2fd6b8822d0295f0e38e230e5ee730/cmd/proc.go#L31):
```go
const (
	DefaultProcRefreshRate = 3000
)
```

## Splitting flag initialization to multiple lines

A flag initialization such as:
```go
procCmd.Flags().Int32P("refresh", "r", DefaultProcRefreshRate, "Process information UI refreshes rate in milliseconds greater than 1000")
```
could be re-written to:
```go
procCmd.Flags().Int32P(
    "refresh",
    "r",
    DefaultProcRefreshRate,
    "Process information UI refreshes rate in milliseconds greater than 1000",
)
```

### PR DESCRIPTION
**Pull request by souvikmaji**
refers to  #65. this PR contains changes from #67 . So please first merge that PR to resolve #64 before considering this.

### CODE REVIEW

**Review by Gituser143 — APPROVED** [review]
This looks fine to me @souvikmaji, considering they're just simple type changes. I'll merge the PR after another approval!

**Comment by Gituser143** [comment]
Fixes #65

**Comment by MadhavJivrajani** [comment]
@souvikmaji please feel free to try out any of the other issues that you'd like and we'll be more than happy to help you out with any of them!

**Pull request closed without merging**
